.github/ISSUE_TEMPLATE/release-team-lead-progress.md

```diff
@@ -0,0 +1,271 @@
+  - Example: 1.25, 1.24 -->
+- [ ] Begin paying attention to CI signal, as it may begin degrading soon after the prior release is cut and any slips must be caught and rectified promptly.
+- [ ] Meet your Shadows and create a communication channel with them. Establish expectations and share out work - delegate!
+- [ ] Request review of this document by the Release Team Lead shadow(s). The shadow(s) should also take all actions in this document around joining groups and requesting access permissions.
+- [ ] Update the SIG Release groups in the `k/k8s.io/groups/sig-release/groups.yaml` with the following:
+  - [ ] Add Lead and Lead shadows to members of `k8s-infra-release-viewers`
+  - [ ] Add EA, Lead, Lead shadows, comms lead, comms shadows to members of `release-comms`
+  - [ ] Add EA, Lead and Lead shadows to members of `release-managers`
+  - [ ] Add EA, Lead and Lead shadows to managers of `release-team`
+  - [ ] Add Role Leads and Role shadows to members of `release-team`
+  - [ ] Add Role shadows and Lead shadows to members of `release-team-shadows`
+  - [ ] Add EA to manager of release-team-shadows, if EA is owner of `release-team-shadows` already then add Lead to manager of `release-team-shadows`
+  - [ ] Add Bug Triage Lead and shadows to `k8s-infra-rbac-triageparty-release`
+- [ ] Ensured that there is a branch manager available for cutting x.y.0-alpha.1
+
+<!-- - [ ] Assist the Enhancements Lead in collecting planned work from SIGs -->
+- [ ] Discussed and scheduled a weekly Release Team meetings on a day that is most acceptable to the team. Invite the `kubernetes-sig-release` group.
+- [ ] Poll Release Team membership and schedule a weekly alternate meeting to better enable more attendance outside of the Americas.
+- [ ] Major release cycle events have been added to the Kubernetes Release Calendar with one week in advance reminders set. (defer to the [handbook for more information](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/release-team-lead#working-with-the-release-team-calendar))
+<!-- - [ ] Prepare for x.y.0-alpha.2 release and that [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) tests are all green -->
+<!-- - [ ] Identified any other dependent ecosystem projects that need release coordination -->
+- [ ] Checked in with ci-signal and branch managers if x.y.0-alpha.2 is ok to be released and [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) tests are all green
+
+
+**A week before Enhancements Freeze:**
+- [ ] Remind the community about Enhancements Freeze reminder send out. Example email: [1.26](https://groups.google.com/a/kubernetes.io/g/dev/c/pJZC2cnkeJ8); It may also be useful to send a short version to the `#sig-release` and `#chairs-and-techleads` slack channels referencing to the k-dev email.
```

Comment:
```suggestion
- [ ] Remind the community about Enhancements Freeze reminder send out. Example email: [1.26](https://groups.google.com/a/kubernetes.io/g/dev/c/pJZC2cnkeJ8); It may also be useful to send a short version to the `#sig-release` and `#chairs-and-techleads` Slack channels referencing to the k-dev email.
```